CORTXTestimonials.md

```diff
@@ -0,0 +1,35 @@
+# Testimonials
+
+The CORTX project is world's only 100% Open Source Mass-Capacity Optimized Object Store solution. We are comitted to creating an eco-system for research and development around storage that is better, more economical, and changed the way the world looks at data and storage. Read what our community members who have deployed CORTX in their environments have to say.
+
+### Jacques-Charles Lafoucriere, Program Manager, CEA
```

Comment:
Format should be like:

### CEA

"Quote"

*   Jacques-Charles Lafourcriere, Program Manager